phpunit.xml.dist

```diff
@@ -1,15 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<phpunit colors="true" bootstrap="vendor/autoload.php">
-    <testsuites>
```

Comment:
Are you sure that you don't need `<testsuites>` element? What do you think @Nyholm ?

Reply:
phpunit itself generates it like that. Try running `phpunit --generate`.

Also, see how the test suite is recognized, and how it is not when I make a typo:

```
phpunit --testsuite MatthiasSymfonyConfigTest
PHPUnit 7.5.1 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.2.14
Configuration: /home/greg/dev/SymfonyConfigTest/phpunit.xml.dist

.......................................                           39 / 39 (100%)

Time: 66 ms, Memory: 10.00MB

OK (39 tests, 65 assertions)

phpunit --testsuite MatthiasSymfonyConfigTes 
PHPUnit 7.5.1 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.2.14
Configuration: /home/greg/dev/SymfonyConfigTest/phpunit.xml.dist



Time: 52 ms, Memory: 10.00MB

No tests executed!

```

Reply:
Didn't know 👍

Reply:
Me neither until yesterday ^^